mypy/test/testfinegrained.py

```diff
@@ -71,6 +71,9 @@ def run_case(self, testcase: DataDrivenTestCase) -> None:
             a.append('==')
             a.extend(new_messages)
 
+        # Normalize paths in test output (for Windows).
+        a = [line.replace('\\', '/') for line in a]
```

Comment:
I guess this doesn't really matter as much, but I am always hesitant to use unix forward slash on Windows, I think it is okay here as long as it is internal and the file itself is never written to disk, but to avoid bugs I always favor the platform specific separator. Just a thought.